requirements_test.txt

```diff
@@ -1,9 +1,11 @@
 dr-usertool
 pytest
 pytest-runner
+mock
```

Comment:
please always pin versions

Reply:
sorry this is a leftover from previous iterations